src/gobexport/requests.py

```diff
@@ -52,28 +57,26 @@ def _exec(method, url, headers, **kwargs):
             print(f"Request exception: '{str(e)}'")
             print(f"Retry in {_RETRY_TIMEOUT} seconds, retries left: {_MAX_TRIES - n_tries}")
             time.sleep(_RETRY_TIMEOUT)
-            # Update headers because access token might be expired
-            headers = _updated_headers(url=url, headers=headers)
 
     request = ', '.join([f"{k}='{v}'" for k, v in kwargs.items()])
     raise APIException(f"Request '{request}' failed, tried {n_tries} times")
 
 
 def _updated_headers(url, headers=None, secure_user=None):
```

Comment:
Why secure_user = None, will raise assertion.

Reply:
Probably a nice idea to elaborate that in the docstring.